engine/orchestrator/score-orchestrator-impl/src/main/java/io/cloudslang/orchestrator/services/ScoreDeprecatedImpl.java

```diff
@@ -86,6 +107,32 @@ public Long reTrigger(SystemContext newSystemContext, byte[] executionObj) {
         return newExecutionId;
     }
 
+    private ExecutionPlan cloneExecutionPlanWithoutSteps(ExecutionPlan executionPlan) {
+        ExecutionPlan newExecutionPlan = new ExecutionPlan();
+        newExecutionPlan.setBeginStep(executionPlan.getBeginStep());
+        newExecutionPlan.setFlowUuid(executionPlan.getFlowUuid());
+        newExecutionPlan.setLanguage(executionPlan.getLanguage());
+        newExecutionPlan.setName(executionPlan.getName());
+        newExecutionPlan.setSubflowsUUIDs(executionPlan.getSubflowsUUIDs());
+        newExecutionPlan.setSysAccPaths(executionPlan.getSysAccPaths());
+        newExecutionPlan.setWorkerGroup(executionPlan.getWorkerGroup());
+        return newExecutionPlan;
+    }
+
+    private Map<String, Object> setNavigationNextStep(ExecutionStep executionStep, Long nextStep){
```

Comment:
we can name it getNavigationNextStep, now that it returns newPreconditionNavigationData